packages/coil-extension/src/util/currencyFormatting.ts

```diff
@@ -16,7 +16,7 @@ Object.entries(symbols).forEach(([k, v]) => {
 
 export function formatAmount(
   amount: Amount,
-  { precision = 2 }: FormatOptions = {}
+  { precision = 0 }: FormatOptions = {}
```

Comment:
@tuxd 
Bad intelligence, bad bad intelligence!

Reply:
We should just specify precision where we want it used, e.g. `formatAmount(amt, {precision: 0})`